setup.cfg

```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = pretrain
+version = 0.0.1
```

Comment:
Consider starting with 0.1.0 to comply with [semver](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase)

Reply:
yep, good point, will change before this merges